modules/test/playwright/helpers/HeadlessAdminUserApiHelper.ts

```diff
@@ -19,4 +19,82 @@ export class HeadlessAdminUserApiHelper {
 			`${this.apiHelpers.baseUrl}${this.basePath}/sites/by-friendly-url-path/${friendlyUrlPath}`
 		);
 	}
+
+	async postAccount(
+		name: string,
+		type: string,
+    ) {
+		const postAccount = await this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/accounts`,
+			{
+				name: name,
+				type: type,
+    		}
+		);
+
+		return postAccount;
+	}
+
+	async postAccountVirtualInstance(
+		name: string,
+		type: string,
+		baseUrl: string
+    ) {
+		const postAccount = await this.apiHelpers.post(
+			`${baseUrl}${this.basePath}/accounts`,
+			{
+				name: name,
+				type: type,
+    		}
+		);
+
+		return postAccount;
+	}
+
+	async postUserAccount(
+		alternateName: string,
+		currentPassword: string,
+		emailAddress: string,
+		familyName: string,
+		givenName: string,
+		password: string,
+    ) {
+		const postUserAccount = await this.apiHelpers.post(
+			`${this.apiHelpers.baseUrl}${this.basePath}/user-accounts`,
+			{
+				alternateName: alternateName,
+				currentPassword: currentPassword,
+				emailAddress: emailAddress,
+				familyName: familyName,
+				givenName: givenName,
+				password: password,
+    		}
```

Comment:
Make sure to provide only the bare minimum for all the API calls.

For example, if the POST can be done with only password and givenName (only an example, probably not true), then you should only use those two properties.

Reply:
**ALL** your post calls need to have the ability to do the post call with default parameters. See one of my classes for example from LPD-4173